src/System.Management.Automation/engine/InternalCommands.cs

```diff
@@ -295,12 +295,12 @@ protected override void ProcessRecord()
                                 else
                                 {
                                     // we write null out because:
-                                    // PS C:\> �$null | ForEach-object {$_.aa} | ForEach-Object {$_ + 3}�
+                                    // PS C:\> "$null | ForEach-object {$_.aa} | ForEach-Object {$_ + 3}"
```

Comment:
The original file before my fixes contained a `0x93` byte at the start of the comment line, and a `0x94` byte at the end of the comment line. In CP-1252, those are opening and closing curly quotes, so I replaced them with straight quotes since that was the closest thing to the original intent of whoever wrote that comment.

But since that comment's original intent was apparently *incorrect*, I'll be happy to remove the quotes.